### PR TITLE
Add workspace encryption guide

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Adds guide on how to use `plural crypto backups`

### CODE REVIEW

File: pages/getting-started/manage-git-repositories/workspace-encryption.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Out of the box, Plural encrypts all configuration in your Plural workspace and stores an encryption key on your local machine.
+This means that if anyone gets a copy of your Plural Git repository/workspace, all sensitive information will appear encrypted to them.
+
+If you are changing machines or collaborating with colleagues, you'll need to know how to create and restore from the workspace
+encryption backups.
+
+## Your encryption key
+
+When you initialize your Plural repository, Plural creates a directory in your home director called `.plural`. Within this directory
```
Comment (swoodward90) [inline pages/getting-started/manage-git-repositories/workspace-encryption.md:14]: nit: home "directory"

File: pages/getting-started/manage-git-repositories/workspace-encryption.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+is a file called `key`, which is your local encryption key. This key is required to decrypt any sensitive configuration within your
```
Comment (swoodward90) [inline pages/getting-started/manage-git-repositories/workspace-encryption.md:15]: is it documented somewhere how to do this? is that the crypto unlock command?

Comment (michaeljguarino) [reply pages/getting-started/manage-git-repositories/workspace-encryption.md:15]: it's autogenerated, but if you want to import one, you can use the `cat /path/to/key | plural crypto import` command

File: pages/getting-started/manage-git-repositories/workspace-encryption.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Plural workspace.
+
+## Encryption backup operations
+
+### Create workspace backup
+
+To create a backup for your local `key`, run `plural crypto backups create`. In the case that you lose your local encryption key,
```
Comment (michaeljguarino) [inline pages/getting-started/manage-git-repositories/workspace-encryption.md:22]: can we put these commands into code blocks?  It adds the copy/paste button

File: pages/getting-started/manage-git-repositories/workspace-encryption.md (hunk 1 above)
Comment (avaidyanatha) [reply pages/getting-started/manage-git-repositories/workspace-encryption.md]: done

File: pages/getting-started/manage-git-repositories/workspace-encryption.md (hunk 2 above)
Comment (avaidyanatha) [reply pages/getting-started/manage-git-repositories/workspace-encryption.md:15]: added a note about this

File: pages/getting-started/manage-git-repositories/workspace-encryption.md (hunk 3 above)
Comment (avaidyanatha) [reply pages/getting-started/manage-git-repositories/workspace-encryption.md]: yep, done

**Pull request merged**